sky/skypilot_config.py

```diff
@@ -13,7 +13,7 @@
 
 To pop a nested-key config:
 
-  >> config_dict = skypilot_config.pop_nested(('auth', 'some_key'))
+  >> config_dict = skypilot_config.set_nested(('auth', 'some_key'), value)
```

Comment:
Update L14, and possibly check the rest of the doc?

Reply:
Done. Thanks!